mycroft/configuration/lang.py

```diff
@@ -19,4 +19,13 @@
 lingua-franca (https://github.com/mycroftai/lingua-franca) selected language
 """
 
-from lingua_franca.lang import set_active_lang, get_active_lang
+from lingua_franca.lang import set_default_lang as _set_default_lf_lang
+
+
+def set_default_lf_lang(lang_code="en-us"):
```

Comment:
why put `lf` in the function name?  one of the points of wrapping libraries like this is to abstract out the dependency on said library.

Reply:
I wanted to make it clear that it's not the default language for all of Mycroft - so the full name would be something like:
`set_default_formatting_and_parsing_language()`

We could make it:
`set_default_util_lang()`
?

That's more correct than `set_default_lang()`, but less correct than `set_default_lingua_franca_lang()`

Reply:
FWIW, I reckon this will eventually be the default language for all of Mycroft, in that runtime TTS and STT swapping will either go in this function.

```python
def set_default_lang(lang: str):
    lingua_franca.set_default_lang(lang)
    mycroft.stt.something(lang)
    ...
```

which may not be an argument for calling it `set_default_lang()` *now*, but it might excuse the decision to give this a name directly reflective of what it wraps (like this one.)

Reply:
However, I would like to discourage direct imports from `lingua_franca.lang`, as the "user"-facing functions are all exposed in `lingua_franca`.

Reply:
Agreed - I hope for this method to be short-lived and deprecated as soon as we can set all systems languages.

Also thanks for pointing out the import location - have updated that.